NEWS.md

```diff
@@ -1,3 +1,7 @@
+1.38.0
+
+* Fix `heroku run` exit code not returning on migrations
```

Comment:
Would something like "Successful deploys with failing DB migrations on Heroku bugfix" explain this better?